### PR TITLE
Implement Poke Ball messages

### PR DESCRIPTION
**Pull request by Joimer**

### CODE REVIEW

**Comment by Joimer** [comment]
Do not merge in this pull request until its counterpart on the server is ready.

**Comment by TheImmortal** [comment]
It should only show the message for special Poke Balls and not the regular Pokeball imo.

**Comment by Joimer** [comment]
That's a good idea actually, as it will make the fact of a special Poke Ball more important. However, the idea is in the future to have a custom sprite for each Poke Ball and be able to choose it from teambuilder. What do you think, @Zarel ?

File: js/battle.js [hunk 1]
```diff
@@ -1563,10 +1565,11 @@ var Side = (function () {
 			pokemon.copyVolatileFrom(this.lastPokemon);
 		}
 
+		var ballName = Tools.getItem(pokemon.pokeball).name || 'Poke Ball';
 		if (pokemon.side.n === 0) {
-			this.battle.message('Go! ' + pokemon.getFullName() + '!');
+			this.battle.message('Go! ' + pokemon.getFullName() + '! (In a ' + ballName + ')');
```
Comment (Zarel) [inline js/battle.js]: I'd say only say "In a ____" if it's relevant (not just a Poké Ball).

**Comment by Zarel** [comment]
lol, turns out you guys were already talking about it.

But yes, eventually they can be selectable from the teambuilder, sre.

**Comment by Zarel** [comment]
This doesn't seem particularly useful, without a way for the server to tell the client which pokeball an opponent is using.

**Comment by Joimer** [comment]
Indeed, that's why I commented "Do not merge in this pull request until its counterpart on the server is ready". I have to struggle with team parser and make it sure it doesn't break for people using an old client.

**Comment by Zarel** [comment]
To clarify, the counterpart on the server isn't ready yet, is it? You're not waiting on me to do anything, are you?

**Comment by Joimer** [comment]
Nothing done on server yet.

**Comment by Zarel** [comment]
This is unmaintained and currently has a merge conflict and doesn't seem to have enough work in it to merit remaining open; closing.

**Comment by Zarel** [comment]
Zarel/Pokemon-Showdown#1355 remains open to continue tracking the feature involved.

**Pull request closed without merging**